requirements.txt

```diff
@@ -1,5 +1,5 @@
 psycopg2==2.6.2
-https://github.com/datamade/django-councilmatic/zipball/rtf-convert
+https://github.com/datamade/django-councilmatic/zipball/master
 django-councilmatic-notifications<0.2
 pysolr===3.6.0
 django-haystack==2.6.1
```

Comment:
Oh...I merged the `log` branch this morning, which updates the requirements. Could you get your branch up-to-date with that change?

Reply:
https://github.com/datamade/nyc-council-councilmatic/commit/b6651a9570d8dad5c8af2c0555268215fa5bb9af